### DISCUSSION TITLE
Error when save code

### DISCUSSION THREAD

**Discussion by ghost** [community/community#124417] [Pull Requests]
### Select Topic Area

Bug

### Body

There was an error committing your changes: File could not be edited

**Comment by ghost** [comment]
> ### Select Topic Area
> Bug
> 
> ### Body
> There was an error committing your changes: File could not be edited
![image 1](https://github.com/community/community/assets/167439052/2bebd587-cabc-4a5f-bf34-a3c488dbfc8f)

**Comment by adnanbaqi** [comment]
Consider these, 
edit :- if found useful mark it as answered 

Permissions Issue: You might not have the necessary permissions to edit the file. This is common in shared or protected environments.

File Lock: The file could be locked by another process or user. If the file is in use by another program or if it is open in another instance, it might prevent you from editing it.

File Corruption: If the file has become corrupted, it may not be editable until it's repaired or restored from a backup.

Version Control System Restrictions: If the file is part of a version control system (like Git), there could be conflicts or restrictions (like pre-commit hooks) that are preventing the changes from being committed.

Storage Issues: There could be issues with the storage device where the file is located, such as being read-only or having no available space.

**Comment by Ushan-Dilusha** [comment]
please check your permission on this repository.
or Branch protection rules
